src/platform/STM32/timer_stm32h7xx.c

```diff
@@ -104,16 +102,19 @@ const timerHardware_t fullTimerHardware[FULL_TIMER_CHANNEL_COUNT] = {
     DEF_TIM(TIM4, CH2, PB7, 0, 0, 0),
     DEF_TIM(TIM4, CH3, PB8, 0, 0, 0),
     DEF_TIM(TIM4, CH4, PB9, 0, 0, 0),
-
     DEF_TIM(TIM12, CH1, PB14, 0, 0, 0),
     DEF_TIM(TIM12, CH2, PB15, 0, 0, 0),
 
+    DEF_TIM(TIM8, CH2N, PB0, 0, 0, 0),
+    DEF_TIM(TIM8, CH3N, PB1, 0, 0, 0),
+    DEF_TIM(TIM8, CH2N, PB14, 0, 0, 0),
+    DEF_TIM(TIM8, CH3N, PB15, 0, 0, 0),
+
```

Comment:
CHxN are not that useful. Removing validates build, but order or other added pins should be verified.

```suggestion
```

Reply:
I validated that the added pins do not change the definition order. Yeah I'm confused why these 4 are causing build failures?